### ISSUE TITLE
Duplicates - by looking at the name

### ISSUE THREAD

**Issue by sstemann** [NCATSTranslator/Feedback#145]
Steps to recreate:  

1.  Run the CI UI for cystic fibrosis:  https://ui.ci.transltr.io/results?l=Cystic%20Fibrosis&t=0&q=f95f5f8f-39e9-4a53-9de6-59a2839f76e2
2. Do an entity search for Azithromycin
3. Notice two entries
![image](https://user-images.githubusercontent.com/38321826/222554622-5b887b71-0b84-4a4e-b72b-68e2e7262bff.png)

4. Investigate in ARAX UI https://arax.ncats.io/?r=f95f5f8f-39e9-4a53-9de6-59a2839f76e2
5. Determine that it looks like BTE and Improve were merged, ARAX result is the standalone (ARAGORN was stuck running for the entirety of this investigation) 
![image](https://user-images.githubusercontent.com/38321826/222555328-20539404-ee65-42f6-857a-6057f6b315d3.png)

6.  I realize this is by design, but when one result is score 89.8 and the other is 40, it seems like we are giving conflicting answers about the "same" result.

7.  This is probably wrapped up in merging and ranking, but maybe we should also consider these cases for when we are determining which attributes are displayed. 

9. In this result set, i think Ceftazidime, Ethinyl estradiol, Fenretinide, Genistein, Gentamicin, Levofloxacin, Rifampin (one is 73.8 the other is 7), Somatropin, Zoledronic acid

**Comment by cbizon** [comment]
These 2 pubchems seem to be different by chirality.  For what it's worth, the pubchem in the image on the left doesn't claim to be azythromycin, nor does the chembl it's linked to.  So I think that one way of looking at this is that the chemical on the left is erroneously labeled.  When the ARS normalizes using node normalizer and gets the labels, these two at least will get different labels so won't look the same.

Though of course there is a way of thinking that maybe this level of hair splitting isn't appropriate and we should be more flexible in which chemicals are considered the same, probably by allowing chirality to be a conflation option in normalizer.

**Comment by sstemann** [comment]
@MarkDWilliams could you chime in here?

**Comment by MarkDWilliams** [comment]
Looking at CHEMBL.COMPOUND:CHEMBL429938, it doesn't look to me like it corresponds to Azithromycin.  Looking at it in Chembl, I get this https://www.ebi.ac.uk/chembl/compound_report_card/CHEMBL4299381/ which doesn't seem to have any resemblance to azithromycin.  I think @cbizon is correct that the one on the left has been mistakenly grouped with the others by Arax.  Node Normalizer does not seem to make this conflation.  As to the question of chirality, I would leave that to people with more knowledge in chemistry.  I know there are some cases in which chirality makes a great deal of difference, but I don't know how often and how often our data sources make a distinction.

**Comment by sstemann** [comment]
based on the 3/7 Standup, ARAX is going to review their specialized normalizer (the example was Fludrocortisone and Fludrocortisone Acetate both being displayed as Fludrocortisone).

Perhaps it is related here as well @dkoslicki

**Comment by dkoslicki** [comment]
Yes, this is being tracked at: https://github.com/RTXteam/RTX/issues/1971

**Comment by amykglen** [comment]
the conflation of Fludrocortisone with Fludrocortisone Acetate has been fixed in ARAX dev/ci instances:

https://arax.ci.transltr.io/?term=fludrocortisone
https://arax.ci.transltr.io/?term=fludrocortisone%20acetate

think this issue can be closed?
